### PR TITLE
Summary component

### PR DESCRIPTION
**Pull request by dleard**
Re-implement summary component
  - fixed nav bug
  - simplified jsonSchema forms for summary
  - reimplemented thank you page & submit button

### CODE REVIEW

File: app/containers/Applications/ApplicationWizardConfirmation.tsx [hunk 1]
```diff
@@ -11,18 +12,25 @@ import ApplicationWizardConfirmationCardItem from './ApplicationWizardConfirmati
 export const ApplicationWizardConfirmationComponent = props => {
```
Comment (matthieu-foucault) [inline app/containers/Applications/ApplicationWizardConfirmation.tsx]: typed props please

File: app/containers/Applications/ApplicationWizardConfirmationCardItem.tsx [hunk 2]
```diff
@@ -1,125 +1,83 @@
...
-        </td>
-        <td style={{width: '50%'}}>{value}</td>
-      </>
-    );
+  const CUSTOM_FIELDS = {
+    TitleField: props => (
+      <h3>{props.title === formJson.schema.title ? null : props.title}</h3>
+    ),
+    StringField: props => (
+      <>: {props.formData ? props.formData : '[No Data Entered]'}</>
```
Comment (matthieu-foucault) [inline app/containers/Applications/ApplicationWizardConfirmationCardItem.tsx]:
```suggestion
      <>: {props.formData ? props.formData : <i>[No Data Entered]</i>}</>
```

File: app/containers/Applications/ApplicationWizardConfirmationCardItem.tsx [hunk 3]
```diff
@@ -1,125 +1,83 @@
...
+            showErrorList={false}
+            fields={CUSTOM_FIELDS}
+            customFormats={customFormats}
+            schema={schema}
+            uiSchema={uiSchema}
+            ObjectFieldTemplate={FormObjectFieldTemplate}
+            formData={query}
+          >
+            <div style={{display: 'none'}}>
+              <Button type="submit">Submit</Button>
```
Comment (matthieu-foucault) [inline app/containers/Applications/ApplicationWizardConfirmationCardItem.tsx]: is the button needed? Can't you just put an empty `span` or fragment?

File: app/containers/Forms/Form.tsx [hunk 4]
```diff
@@ -3,11 +3,12 @@ import {graphql, createFragmentContainer} from 'react-relay';
 import JsonSchemaForm, {
   IChangeEvent,
   ErrorSchema,
-  UiSchema,
+  // UiSchema,
```
Comment (matthieu-foucault) [inline app/containers/Forms/Form.tsx]:
```suggestion
```

File: app/containers/Applications/ApplicationWizardConfirmation.tsx [hunk 5]
```diff
@@ -73,11 +73,12 @@ export default createFragmentContainer(ApplicationWizardConfirmationComponent, {
...
           edges {
             node {
-              formResult
+              id
+              ...ApplicationWizardConfirmationCardItem_formResult
             }
           }
         }
-        orderedFormResults {
+        applicationStatusesByApplicationId(orderBy: CREATED_AT_DESC) {
```
Comment (matthieu-foucault) [inline app/containers/Applications/ApplicationWizardConfirmation.tsx]: Use `applicationStatus` to get the latest status

File: app/containers/Applications/ApplicationWizardConfirmationCardItem.tsx [hunk 6]
```diff
@@ -1,125 +1,83 @@
...
+  Props
+> = ({formResult}) => {
+  const {formJsonByFormId} = formResult;
+  const query = formResult.formResult;
+  const {formJson} = formJsonByFormId;
+  const {
+    schema,
+    uiSchema,
+    customFormats
+    // CustomFormatsErrorMessages = {}
```
Comment (matthieu-foucault) [inline app/containers/Applications/ApplicationWizardConfirmationCardItem.tsx]:
```suggestion
```

File: app/containers/Applications/ApplicationWizardConfirmationCardItem.tsx (hunk 3 above)
Comment (dleard) [reply app/containers/Applications/ApplicationWizardConfirmationCardItem.tsx]: just a display none div is enough apparently. I thought we'd need the button to override the default button, but nope!

**Pull request merged**